src/index.js

```diff
@@ -404,7 +408,23 @@ let readScores = (workbook) => {
                 jamNumber = sheet[XLSX.utils.encode_cell(jamAddress)]
 
                 // if we're out of jams, stop
-                if (jamNumber == undefined || jamNumber.v == undefined){break}
+                if (jamNumber == undefined || jamNumber.v == undefined) {
```

Comment:
This is the actual thing I'm trying to add.

Reply:
If I'm reading this right, this commit throws a breaking error if a nonsequential jam number is encountered.  I just updated the main branch on Wednesday to add an error to the report when this happens, rather than giving up entirely, which would be my preference.

Reply:
You appear to have merged this anyway though? I can't see where code for nonsequential jam numbers exists in here or in master, but as noted, I am terrible at using git and github.

Reply:
I didn't merge this into the main branch, rather, I resolved a conflict with this pull request by merging part of the main branch into the pull request.  (Specifically, the version number.)  At least - I *think* that's what I did.  I'm not a wiz at this thing either.

The code to check for out of sequence numbers was already present at line 426 in index .js, but it broke if you skipped a number entirely.  Commit f03f222a7ccbdfd8890ecafa5f08a63967a6fcd2 was a fix to that bug.